cmd/flags/flags.go

```diff
@@ -68,7 +70,26 @@ func AddNoClobberFlag(cmd *cobra.Command) {
 }
 
 func AddPathFlag(cmd *cobra.Command) {
-	cmd.Flags().StringP("path", "p", "", "Path to project. Can be any subdirectory of the project, but it must be a git project")
+	cmd.Flags().StringP("path", "p", "", "Path to project. Can be any subdirectory of the workspace root")
+}
+
+func AddRootPathFlag(cmd *cobra.Command) {
+	cmd.Flags().StringP("rootpath", "", "", "Path to workspace root. Defaults to the folder detected by git as the workspace root")
```

Comment:
Either this should be root-path (and the viper.GetString calls be updated) or we need to update the example in the readme

Reply:
I prefer "rootpath" without the hyphen because "root-path" causes the associated env-var to be "TFD_ROOT-PATH" and env-vars with hyphens in their name get dropped by (real/bourne) `sh`

Reply:
Gah, more viper/cobra weirdness. Fair enough